sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/SparkMetadataOperationSuite.scala

```diff
@@ -182,4 +182,31 @@ class SparkMetadataOperationSuite extends HiveThriftJdbcTest {
       checkResult(metaData.getTableTypes, Seq("TABLE", "VIEW"))
     }
   }
+
```

Comment:
Can you add some tests for the usage and the function class name?

Reply:
I have added some tests:
```scala
ssert(rs.getString("REMARKS").startsWith(s"${functionName(i)}("))
assert(rs.getString("SPECIFIC_NAME").startsWith("org.apache.spark.sql.catalyst"))
```
Do you think we need to assert more details?

Reply:
Would be nice to run a ```DESCRIBE function``` statement and then compare the results.

Reply:
Done:
https://github.com/apache/spark/pull/25252/files#diff-1cde3e024f639585984b678383450789R212-R223